test/unit/annotation_spec.js

```diff
@@ -678,7 +678,7 @@ describe("annotation", function () {
       expect(data.creationDate).toEqual("D:20180523");
       expect(data.modificationDate).toEqual("D:20190523");
       expect(data.color).toEqual(new Uint8ClampedArray([102, 102, 102]));
-      expect(data.hasPopup).toEqual(false);
+      expect(!!data.popupRef).toEqual(false);
```

Comment:
How about testing this explicitly instead?
```suggestion
      expect(data.popupRef).toEqual(null);
```